.github/workflows/simulation_test.yml

```diff
@@ -55,23 +77,27 @@ jobs:
 
     - name: Install NEURON
       run: |
-        sudo apt-get install flex libfl-dev bison ninja-build
-        pip install -U pip setuptools
-        pip install "cython<3" pytest sympy
         export SONATAREPORT_DIR=$(pwd)/libsonatareport/build/install
+        git clone https://github.com/BlueBrain/nmodl.git --depth=1
         git clone --branch="${NEURON_BRANCH:-master}" https://github.com/neuronsimulator/nrn.git --depth=1
-        cd nrn
-        mkdir build && cd build
-        cmake -G Ninja -DPYTHON_EXECUTABLE=$(which python) -DCMAKE_INSTALL_PREFIX=$(pwd)/install -DNRN_ENABLE_MPI=ON -DNRN_ENABLE_INTERVIEWS=OFF \
-          -DNRN_ENABLE_CORENEURON=ON -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCORENRN_ENABLE_REPORTING=ON -DCMAKE_PREFIX_PATH=$SONATAREPORT_DIR ..
-        cmake --build . --parallel
-        cmake --build . --target install
+        python -m pip install --upgrade pip -r nrn/nrn_requirements.txt
```

Comment:
Is it needed to install the nmodl dependencies ? We don't do that when building the docker container and it seems working.